pkgs/by-name/ja/jart-jsoncpp/double_conversion.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 7a564de..f7c1f06 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -6,24 +6,13 @@ option(JSON_CPP_BUILD_TESTS "Enable building tests" ON)
+ set(CMAKE_CXX_STANDARD 11)
+ set(CMAKE_CXX_STANDARD_REQUIRED ON)
+ 
+-# Add double-conversion library
+-add_library(double-conversion STATIC
+-    double-conversion/bignum.cc
+-    double-conversion/bignum-dtoa.cc
+-    double-conversion/cached-powers.cc
+-    double-conversion/double-to-string.cc
+-    double-conversion/fast-dtoa.cc
+-    double-conversion/fixed-dtoa.cc
+-    double-conversion/string-to-double.cc
+-    double-conversion/strtod.cc
+-)
+-target_include_directories(double-conversion PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
++find_package(double-conversion CONFIG REQUIRED)
+ 
+ # Main JSON library
+-add_library(json STATIC
++add_library(json SHARED
```

Comment:
I think you can have neither `STATIC` nor `SHARED`, and then it will just do the default, so as not to break `pkgsStatic`.

Reply:
done.